activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeTaskAssigneeTest.java

```diff
@@ -0,0 +1,445 @@
+        assertThat(task.getStatus()).isEqualTo(Task.TaskStatus.ASSIGNED);
+
+
+    }
+
+    @Test
+    @WithUserDetails(value = "salaboy", userDetailsServiceBeanName = "myUserDetailsService")
+    public void bCreateCheckTaskCreatedForSalaboyFromAnotherUser() {
+
+        // the target user should be able to see the task as well
+        Page<Task> tasks = taskRuntime.tasks(Pageable.of(0,
+                50));
+
+        assertThat(tasks.getContent()).hasSize(1);
+        Task task = tasks.getContent().get(0);
+
+        String authenticatedUserId = securityManager.getAuthenticatedUserId();
+        assertThat(task.getAssignee()).isEqualTo(authenticatedUserId);
+        assertThat(task.getStatus()).isEqualTo(Task.TaskStatus.ASSIGNED);
+
+        Task deletedTask = taskRuntime.delete(TaskPayloadBuilder
+                .delete()
+                .withTaskId(task.getId())
+                .withReason("test clean up")
+                .build());
+
+        assertThat(deletedTask).isNotNull();
+        assertThat(deletedTask.getStatus()).isEqualTo(Task.TaskStatus.DELETED);
+
+        tasks = taskRuntime.tasks(Pageable.of(0,
+                50));
+        assertThat(tasks.getContent()).hasSize(0);
+
+
+    }
+
+
+    @Test
+    @WithUserDetails(value = "garth", userDetailsServiceBeanName = "myUserDetailsService")
+    public void cCreateStandaloneTaskForGroupAndClaim() {
+
+
+        String authenticatedUserId = securityManager.getAuthenticatedUserId();
+        Task standAloneTask = taskRuntime.create(TaskPayloadBuilder.create()
+                                                         .withName("group task")
+                                                         .withGroup("doctor")
+                                                         .build());
+
+        // the owner should be able to see the created task
+        Page<Task> tasks = taskRuntime.tasks(Pageable.of(0,
+                                                         50));
+
+        assertThat(tasks.getContent()).hasSize(1);
+        Task task = tasks.getContent().get(0);
+
+        assertThat(task.getAssignee()).isNull();
+        assertThat(task.getStatus()).isEqualTo(Task.TaskStatus.CREATED);
+
+        Task claimedTask = taskRuntime.claim(TaskPayloadBuilder.claim().withTaskId(task.getId()).build());
+        assertThat(claimedTask.getAssignee()).isEqualTo(authenticatedUserId);
+        assertThat(claimedTask.getStatus()).isEqualTo(Task.TaskStatus.ASSIGNED);
+    }
+
+
+    @Test
+    @WithUserDetails(value = "admin", userDetailsServiceBeanName = "myUserDetailsService")
+    public void dCleanUpWithAdmin() {
+        Page<Task> tasks = taskAdminRuntime.tasks(Pageable.of(0, 50));
+        for (Task t : tasks.getContent()) {
+            taskAdminRuntime.delete(TaskPayloadBuilder
+                    .delete()
+                    .withTaskId(t.getId())
+                    .withReason("test clean up")
+                    .build());
+        }
+
+    }
+//
+//    @Test
```

Comment:
What is the status of these commented tests?

Reply:
@erdemedeiros I think that I've just forgot to remove them.. because they must be covered in another test class

Reply:
ok. I'll remove them so.